xml/System.Security.Cryptography.X509Certificates/X509CertificateCollection.xml

```diff
@@ -975,10 +1023,18 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="netcore-1.0;netcore-1.1;netstandard-1.3;netstandard-1.4;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The object to locate in the collection.</param>
+        <summary>Searches for the specified object and returns the zero-based index of the first occurrence within the collection.</summary>
+        <returns>The zero-based index of the first occurrence of <paramref name="value"/> within the collection, if found; otherwise -1.</returns>
```

Comment:
```suggestion
        <returns>The zero-based index of the first occurrence of <paramref name="value"/> within the collection, if found; otherwise, -1.</returns>
```